### PR TITLE
Replaced deprecated helm repo url at docs :page_facing_up:

### PR DESCRIPTION
**Pull request by markrity**

### CODE REVIEW

File: site/docs/v1.2.0/customize-installation.md [hunk 1]
```diff
@@ -63,17 +63,17 @@ To use a different volume snapshot provider:
 
 1. Add your volume snapshot provider's plugin to Velero (look in [your provider][0]'s documentation for the image name):
 
-    ```bash
-    velero plugin add <registry/image:version>
-    ```
+   ```bash
```
Comment (skriss) [inline site/docs/v1.2.0/customize-installation.md]: could you revert this whitespace change? It causes things to render poorly on the website (see the preview at https://deploy-preview-2209--velero.netlify.com/docs/v1.2.0/customize-installation/ -- note code blocks no longer indented and numbering restarts). You'll need to do the same on the `master` version of this page too.

File: site/docs/v1.2.0/customize-installation.md [hunk 2]
```diff
@@ -63,17 +63,17 @@ To use a different volume snapshot provider:
...
+   ```
 
 1. Add a volume snapshot location for your provider, following [your provider][0]'s documentation for configuration:
 
-    ```bash
-    velero snapshot-location create <NAME> \
-        --provider <PROVIDER-NAME> \
-        [--config <PROVIDER-CONFIG>]
-    ```
+   ```bash
```
Comment (skriss) [inline site/docs/v1.2.0/customize-installation.md]: please also revert this whitespace change (also on `master` version of page).

**Comment by skriss** [comment]
You'll also need to add DCO signoff to your commits to pass CI checks -- see https://github.com/vmware-tanzu/velero/runs/397549375.

**Comment by markrity** [comment]
@skriss I have reverted whitespace changes and fixed DCO. 👍

File: changelogs/unreleased/2209-markrity [hunk 3]
```diff
@@ -0,0 +1 @@
+Replaced deprecated helm repo url at all it appearance at docs.
```
Comment (cpanato) [inline changelogs/unreleased/2209-markrity]: maybe a new line here?

Comment (markrity) [reply changelogs/unreleased/2209-markrity]: I have taken a look at all other unreleased changelogs, they look similar to this, so I didn't get the point of new line?

Comment (skriss) [reply changelogs/unreleased/2209-markrity]: I think it's fine as-is, and to be honest we don't usually require changelogs for docs changes anyway, so don't worry about this one.

**Comment by skriss** [comment]
@carlisia PTAL!

**Comment by carlisia** [comment]
Looking! 👀

**Comment by markrity** [comment]
@carlisia Sure , it sounds reasonable, I haven't seen that page before. 
Will push an update soon.

Also , there is a link to github from that page
> For more details on installing Velero please see the chart readme.

I think , there might be a need in converting that "chart readme" to a documentation page (maybe in my next PR) @carlisia What do you think ?

**Comment by carlisia** [comment]
> I think , there might be a need in converting that "chart readme" to a documentation page (maybe in my next PR) @carlisia What do you think ?

Absolutely! That page was put together in a hurry to get all the helm things up and running. I'd love to see an improvement to that!

File: site/docs/master/basic-install.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ brew install velero
 There are two supported methods for installing the Velero server components:
 
 - the `velero install` CLI command
-- the [Helm chart](https://github.com/helm/charts/tree/master/stable/velero)
+- the [Helm chart](https://github.com/vmware-tanzu/helm-charts)
```
Comment (ashish-amarnath) [inline site/docs/master/basic-install.md]:
```suggestion
- the [Helm chart](https://vmware-tanzu.github.io/helm-charts/)
```

File: site/docs/master/customize-installation.md [hunk 5]
```diff
@@ -87,12 +87,11 @@ If you are installing Velero in Kubernetes 1.13.x or earlier, you need to use `k
 
 ## Additional options
 
-Run `velero install --help` or see the [Helm chart documentation](https://github.com/helm/charts/tree/master/stable/velero) for the full set of installation options.
+Run `velero install --help` or see the [Helm chart documentation](https://github.com/vmware-tanzu/helm-charts) for the full set of installation options.
```
Comment (ashish-amarnath) [inline site/docs/master/customize-installation.md]:
```suggestion
Run `velero install --help` or see the [Helm chart documentation](https://vmware-tanzu.github.io/helm-charts/) for the full set of installation options.
```

File: site/docs/v1.2.0/basic-install.md [hunk 6]
```diff
@@ -45,7 +45,7 @@ brew install velero
 There are two supported methods for installing the Velero server components:
 
 - the `velero install` CLI command
-- the [Helm chart](https://github.com/helm/charts/tree/master/stable/velero)
+- the [Helm chart](https://github.com/vmware-tanzu/helm-charts)
```
Comment (ashish-amarnath) [inline site/docs/v1.2.0/basic-install.md]:
```suggestion
- the [Helm chart](https://vmware-tanzu.github.io/helm-charts/)
```

File: site/docs/v1.2.0/customize-installation.md [hunk 7]
```diff
@@ -87,7 +87,7 @@ If you are installing Velero in Kubernetes 1.13.x or earlier, you need to use `k
 
 ## Additional options
 
-Run `velero install --help` or see the [Helm chart documentation](https://github.com/helm/charts/tree/master/stable/velero) for the full set of installation options.
+Run `velero install --help` or see the [Helm chart documentation](https://github.com/vmware-tanzu/helm-charts) for the full set of installation options.
```
Comment (ashish-amarnath) [inline site/docs/v1.2.0/customize-installation.md]:
```suggestion
Run `velero install --help` or see the [Helm chart documentation](https://vmware-tanzu.github.io/helm-charts/) for the full set of installation options.
```

**Comment by skriss** [comment]
@markrity just checking in to see if you'll be able to make the changes suggested. thanks!

**Comment by skriss** [comment]
@ashish-amarnath @carlisia perhaps we want to merge as-is and one of you can follow up with the changes requested.

**Comment by ashish-amarnath** [comment]
@skriss sounds good. I'll have an PR incoming to address the outstanding comments.

**Pull request merged**